test/common/tests.js

```diff
@@ -971,7 +971,10 @@ module.exports = (sql, driver) => {
             assert.ok(err)
 
             if (isSQLServer2019OrNewer) {
-              expectedMessage = "String or binary data would be truncated in table 'master.dbo.tran_test', column 'data'. Truncated value: 'asdfasdfasdfasdfasdfasdfasdfasdfasdfasdfasdfasdfas'."
+              const config = JSON.parse(require('fs').readFileSync(join(__dirname, '../.mssql.json')))
```

Comment:
Could this be `require(join(__dirname, '../.mssql.json'))`?

Reply:
yup, worked a charm! and that's probably also caching it as a module and reading it just once, too. 👍